openapi/horizon/components/schemas/asyncTransactionSubmissionSchema.yml

```diff
@@ -0,0 +1,35 @@
+components:
+  schemas:
+    AsyncTransactionSubmissionResponse:
+      type: object
+      properties:
+        errorResultXdr:
+          type: string
+          nullable: true
+          description: TransactionResult XDR string which is present only if the submission status from core is an ERROR.
+        tx_status:
+          type: string
+          enum: ["ERROR", "PENDING", "DUPLICATE", "TRY_AGAIN_LATER"]
+          description: Status of the transaction submission.
+        hash:
+          type: string
+          description: Hash of the transaction.
+    Problem:
```

Comment:
I would've figured the `Problem` schema was already defined for other error types. Just want to make sure we're not duplicating it here.

Reply:
I'm curious about this, too.

Along those lines, does anyone know when the last time the Horizon spec has been thoroughly gone through, with an eye towards reducing duplicate definitions and such? My suspicion (perhaps wrongly) is a while or never.

If it would be a benefit, I can find some room in the near-medium term to take a shot.

Reply:
@JakeUrban Yeah I checked the code and weirdly, the Problem object is not documented in any schema yamls.

@ElliotFriend I dont think we have updated this part of the Horizon docs (endpoint definitions) in a long time. We last revamped the admin docs but not this one.

Reply:
Ah I see they are defined here - https://developers-pr540.previews.kube001.services.stellar-ops.com/network/horizon/api-reference/errors/http-status-codes/horizon-specific/transaction-failed